packages/pilat/pilat.1.6/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "Steven De Oliveira <de.oliveira.steven@gmail.com>"
+authors: "Steven De Oliveira <de.oliveira.steven@gmail.com>"
+dev-repo: "git://github.com/Stevendeo/Pilat.git"
+homepage: "https://github.com/Stevendeo/Pilat/"
+synopsis: "Polynomial invariant generator"
+license: "LGPL-2.1-only"
+build: ["sh" "make.sh"]
+install: ["sh" "install.sh"]
+depends: [
+    "ocaml" {>= "4.08.1"}
+    "ocamlfind"
+    "lacaml" {<= "11.0.8"}
+    "zarith" {<= "1.12"}
+    "why3" {<= "1.3.0"}
+    "frama-c" {>= "22.0" <= "25.0"}
+    "conf-gtksourceview"
```

Comment:
```suggestion
    "lacaml" {>= "11.0.8"}
    "zarith" {>= "1.12"}
    "why3" {>= "1.3.0"}
    "frama-c" {>= "22.0"}
```

Reply:
The next version of frama-c may break the plug-in and force me to update the version's upper bound. Is it a good practice not to have upper bounds in general?